tests/test_rooms/test_query_rooms.py

```diff
@@ -0,0 +1,16 @@
+from tests.base import BaseTestCase
+
+from fixtures.room.room_fixtures import (
+    rooms_query,
+    query_rooms_response,
+    room_query_by_id,
+    room_query_by_id_response
+)
+class QueryRooms(BaseTestCase):
+    def test_query_rooms(self):
+        query_rooms = self.client.execute(rooms_query)
+        assert query_rooms == query_rooms_response
```

Comment:
I would prefer the use assertEquals instead of the equal sign, remember more words mean more clarity, less logic for the brain in order to make sense of the code

Reply:
I have taken this into consideration and I agree. I have implemented the requested changes accordingly.